### ISSUE TITLE
Human-Ground residuals may be better if it is placed at the pelvis segment in the examples

### ISSUE THREAD

**Issue by mkjung99** [AnyBody/anymocap#21]
https://github.com/AnyBody/anymocap/blob/master/Examples/Plug-in-gait_MultiTrial_StandingRef/LabSpecificData.any

In this example, Human-Ground residuals are set to be located at the trunk segment.

But for some FDK models, this seems to be less stable than the model which has its resdiuals in the pelvis segment.

So we may have to be careful where we should put this residuals in the examples.

**Comment by melund** [comment]
@mkjung99 You are absolutely right. The reason is that the example was developed together with @mariajnsson for her application at KTH in Sweeden. She wanted to analyze the lower-back forces with the MoCap protocol that didn't include the arms. 

But you are of course right. It makes the model more unstable, so it is not the best starting point for new users. 

Maria: (@mariajnsson) Do you have anything against us changing the example so it better fit the need of most new users?

**Comment by melund** [comment]
To make the model a better starting point for new users I would do the following:

* Remove Trunk muscles. 
* Change Human-Ground residuals from Trunk to Pelvis
* Change the leg model to TLEM2.

Are there any other changes you could think of @mkjung99?

**Comment by mkjung99** [comment]
For my models, in general, it was better in terms of stability to include the Trunk muscles even if the model does not include the arms. 
Maybe it is because that Psoas muscles can be sometimes overloaded easily when the Trunk muscles are removed.
But of course there are some drawbacks. By including the Trunk muscles, the model will be more complex so that it may take more time to run the inverse dynamics analysis.

For me, the best situation is:

- Include Trunk muscles (even the models do not include the arms)
- Change Human-Ground residuals from Trunk to Pelvis
- Change the leg model to TLEM2.

**Comment by mariajnsson** [comment]
@melund and @mkjung99 That is absolutely fine for me if you want to change those things!

**Comment by melund** [comment]
> For my models, in general, it was better in terms of stability to include the Trunk muscles even if the model does not include the arms.

@mkjung99 I have the opposite experience. But we can discuss that later. I will make a pull request with the changes.

**Comment by melund** [comment]
Thanks a lot @mariajnsson. Btw. let me know if you need any help porting your models to the new AMMR 2.0 release.

**Comment by melund** [comment]
@mkjung99 I have added a PR for the changes: #22
